apps/web/pages/gallery/[galleryId]/edit.tsx

```diff
@@ -103,6 +103,13 @@ function EditGalleryPageInner({ queryRef }: EditGalleryPageInnerProps) {
     throw new Error('Username does not exist');
   }
 
+  // ummm shouldn't need to do this but sometimes this component
+  // renders when on another route?! this occurs when navigating
+  // away from the editor
+  if (route !== '/gallery/[galleryId]/edit') {
+    return null;
+  }
```

Comment:
This seems a little sus to me. WIth `AnimatedPresence` every route stays mounted for a short while while the new page is animating in.

We should be able to handle that gracefully and not do stuff like this. What was the user facing issue when this stays mounted?